js/script.js

```diff
@@ -0,0 +1,41 @@
+const taskInput = document.getElementById("input-task");
+const addButton = document.getElementById("add-task");
+const taskContainer = document.getElementById("task-container");
+
+const reg = /[&<>"'/`]/ig;
+
+let taskList = [];
+
+const addTaskHandler = () => {
+    const taskTitle = taskInput.value.replace(reg, "").trim();
+    
+    if (!taskTitle.length) {
+        alert("Please enter your task");
+        return;
+    }
+
+    const task = createTask(taskTitle);
+
+    taskList.unshift(task);
+
+    const taskElement = createTaskElement(task);
+
+    taskContainer.innerHTML = taskElement.outerHTML.concat(taskContainer.innerHTML);
+};
+
```

Comment:
```suggestion
const addTodoHandler = () => {
    const taskTitle = sanitize(taskInput.value).trim();
    if (!taskTitle.length) {
        alert("Please enter a valid text");
        return;
    }
    const task = createTask(taskTitle);
    taskList.unshift(task);
    renderTodoArray(todos);
};

const renderTodos = (todos) => {
    $taskList.innerHTML = "";
    tasks.forEach((todo) => {
        $taskList.appendChild(createTodoElement(todo));
    });
};

```
 a separate render function will help you keep separate the rendering and business logics.

Reply:
okay you keep it.